### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/19.09/java_jvm%E5%9E%83%E5%9C%BE%E6%94%B6%E9%9B%86%E5%99%A8

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#60] [General]
# https://www.xiaogenban1993.com/blog/19.09/java_jvm%E5%9E%83%E5%9C%BE%E6%94%B6%E9%9B%86%E5%99%A8

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/19.09/java_jvm%E5%9E%83%E5%9C%BE%E6%94%B6%E9%9B%86%E5%99%A8

**Comment by WRZ10** [comment]
大哥 你的图片都打不开啦

**Reply by sunwu51** [reply]
这几个月的文章被华为云坑了，数据丢失了。
